custom_components/places/entity.py

```
@@ -0,0 +1,283 @@
+)
+
+if TYPE_CHECKING:
+    from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+    class PlacesUpdateCoordinator(DataUpdateCoordinator[Any]):
+        """Typing stub for the future Places coordinator used by shared entities."""
+
+        config_entry: Any
+        device_info: Any
+
+        def get_attr(self, attr_name: str) -> Any:
```

Ruff (0.15.20)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `get_attr`

(ANN401)